### PR TITLE
Add support for downloading provenance and annotations

### PR DESCRIPTION
**Pull request by Bobfrat**
@birdage @DanielJMaher 

This goes along with https://github.com/asascience-open/ooi-ui-services/pull/362

### CODE REVIEW

**Comment by birdage** [comment]
@Bobfrat there is an issue i think, it looks like you can remove one of the selections and download as CSV (with annotations selected, for example)

>Select Include Provenance, Select Include Annotations, {Deselect Annotations or Deselect Provenance}, Download CSV available

File: ooiui/static/js/views/science/StreamDownloadFormView.js [hunk 1]
```diff
@@ -16,11 +16,40 @@
...
+    'click #annotation-select' : 'onAnnotationsSelect',
   },
   initialize: function() {
     this.render();
   },
+  onAnnotationsSelect: function() {
+    if(this.$el.find('#annotation-select').is(':checked')){
+      this.$el.find("#type-select option[value*='csv']").prop('disabled',true);
+    }else{
+      this.$el.find("#type-select option[value*='csv']").prop('disabled',false);
```
Comment (birdage) [inline ooiui/static/js/views/science/StreamDownloadFormView.js]: @Bobfrat you issue is here, and line 38, we need to check for both

**Comment by birdage** [comment]
@Bobfrat both `if` statements to be as follows
`if((this.$el.find('#provenance-select').is(':checked')) || (this.$el.find('#annotation-select').is(':checked'))){ `

**Comment by Bobfrat** [comment]
@birdage check it out now, those functions were identical after the change so I consolidated into 1

**Comment by birdage** [comment]
@Bobfrat :+1:

**Pull request merged**